src/database/utils/migrationLoader.ts

```
@@ -0,0 +1,105 @@
+
+  /**
+   * Get list of available migration files
+   * @returns Array of migration file paths
+   */
+  static getAvailableMigrations(): string[] {
+    return [
+      'migrations/001_create_high_scores_table.sql',
+      'migrations/001_create_high_scores_table_down.sql'
+    ];
+  }
+} 
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)